Test/comp/CompileWithArguments.dfy

```diff
@@ -1,11 +1,11 @@
 // RUN: %dafny /compile:0 "%s" > "%t"
-// RUN: %dafny /noVerify /compile:4 /compileTarget:cs "%s" --args csharp 1 >> "%t"
-// RUN: %dafny /noVerify /compile:4 /compileTarget:cpp "%s" --args cpp Yipee >> "%t"
-// RUN: %dafny /noVerify /compile:4 /compileTarget:java "%s" --args java heya >> "%t"
-// RUN: %dafny /noVerify /compile:4 /compileTarget:js "%s" --args javascript 2 >> "%t"
+// RUN: %baredafny run --no-verify --target:cs %args "%s" csharp 1 >> "%t"
```

Comment:
What is this `%args` for ?

Reply:
It's because `run` needs to be the first argument, so we can't use %dafny because that places the default arguments for tests _before_ the verb. Alternatively, we could replace `%baredafny run %args` with `%run`

Reply:
Oh, I got it. It has nothing to do with `--args`, it's just the other options that previously were inserted in %dafny.
Would it make sense to name this option `%defaultArgs` or something similar then? At first, I thought it would be the new separators for the arguments. Or you could just move `%args` after `run`